haiku-apps/wordclock/wordclock-0.1.1.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="An imprecise clock"
+DESCRIPTION="WordClock is an imprecise clock that displays the time in words \
+instead of digits. You can change the used font and embed the clock as a \
+Replicant on the Desktop."
+HOMEPAGE="https://github.com/Janus2/WordClock"
+COPYRIGHT="2017 Janus"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="cbb5da9a498ec99a71a68f039042601207eafdd1"
+SOURCE_URI="https://github.com/Janus2/WordClock/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="fe29a3f17467927f88052d9c260b894d8424cf175b633aa2ad22deea2b25abba"
+SOURCE_FILENAME="WordClock-$srcGitRev.tar.gz"
+SOURCE_DIR="WordClock-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	wordclock = $portVersion
+	app:WordClock = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	"
+
+BUILD()
+{
+	make $jobArgs
+	make bindcatalogs
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir
+	make install TARGET_DIR=$appsDir
+
+	addAppDeskbarSymlink $appsDir/WordClock
+}
+
```

Comment:
No need for a blank line at the end of the recipe.
Could you please remote it? Thanks :-)

Reply:
oops. too late... :)

Reply:
Never mind ;-)

Reply:
Just for my education... wasn't it recommended at some time to always have a blank line at the end of files? For people using vim IIRC...

Reply:
Some editors, like Pe (if I'm not mistaken) as well as those which run on a popular OS which uses CRLF-style end-of-lines (instead of LineFeed on Unix-like OSes) do not bother to add anything at the end of the very last line of a text file.
So when you open with vi a file that was saved with those "text editors", vi complains/warns about that by saying: "[noeol]" on Haiku or "[Incomplete last line]" on Linux. Then it adds the LF for you. This means you can "repair" the file just by saving it with vi.
So, for those who don't use vi, adding an empty line forces those "text editors" to add a CRLF+CR (or 2 LF), but I'm not really sure about what these editors add on such cases.

On GitHub, when a file is missing the LF at the last lines, we see a special zero character/glyph which makes it clear that the file is missing a LF there. So it is safe to ask for the deletion of the blank line if this spcial zero char/glyph was not shown :-)

Reply:
Thanks for the explanation.
> saying: "[noeol]" on Haiku

That's almost christmassy... :)